build.gradle

```diff
@@ -291,4 +293,33 @@ project(':vgen') {
       args = clargs.split(',').toList()
     }
   }
+
+  task verifyLicense << {
+    def licenseText = '''/* Copyright 2016 Google Inc
```

Comment:
It would be nice if we could derive the check from a file that was outside of this build file.